ner_v2/detectors/pattern/phone_number/phone_number_detection.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+from ner_v2.detectors.base_detector import BaseDetector
+from ner_v2.detectors.numeral.number.number_detection import NumberDetector
+from language_utilities.constant import ENGLISH_LANG
+import re
+
+
+class PhoneDetector(BaseDetector):
+    """
+    This method is used to detect phone numbers present in text. The phone detector takes into
+    consideration domestic as well as international phone numbers.
+
+    Attributes:
+         text(str): string provided to extract phone numbers detection
+         tagged_text (str): string in which the detected phone numbers are replaced by __<entity_name>__
+         processed_text (str): string in which the detected phone numbers are removed
+         phone (list): list of detected entity values
+         original_phone_text (list): list to store substrings of the text detected as phone numbers
+         tag (str): entity_name prepended and appended with '__'
+    """
+    def __init__(self, entity_name, language=ENGLISH_LANG):
+        """
+        Args:
+            entity_name (str): A string by which the detected numbers would be replaced with
+            on calling detect_entity()
+            language (str, optional): language code of number text, defaults to 'en'
+        """
+        self._supported_languages = NumberDetector.get_supported_languages()
```

Comment:
Which languages do we provide support for?.  We have no dependency on any data or any detector so we can support a range of Phone Numbers (Provided python supports them)
_(As for now, I have initialized the supported languages from the Number Detector just to maintain consistency)_

Please suggest what is the appropriate solution to this.